### PR TITLE
[2971][ADD] commission_disable_regenerate_agent

### PR DESCRIPTION
**Pull request by AungKoKoLin1997**
[2971](https://www.quartile.co/web#view_type=form&model=project.task&id=2971&active_id=2971&menu_id=)

### CODE REVIEW

**Review by yostashiro — COMMENTED** [review]
Sorry to be rather pedantic. Can we please rename the module to account_move_hide_regenerate_agents_button.

**Comment by AungKoKoLin1997** [comment]
> Sorry to be rather pedantic. Can we please rename the module to account_move_hide_regenerate_agents_button.

@yostashiro After communication with kanda san, this module should be for sale and invoice. So, I added the sale order part and change module name to "hide_regenerate_agents_button"

**Comment by yostashiro** [comment]
@AungKoKoLin1997 Thanks, yes that makes sense.  Could we then make it `commission_disable_regenerate_agent`, so that the correlation to the commission modules is clearer.

**Review by kanda999 — APPROVED** [review]
Function review.

File: commission_disable_regenerate_agent/views/account_invoice_views.xml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        <field name="inherit_id" ref="sale_commission.view_order_agent_form_inherit" />
+        <field name="arch" type="xml">
+            <xpath
+                expr="//button[@name='recompute_lines_agents']"
+                position="attributes"
+            >
+                <attribute name="invisible">True</attribute>
+            </xpath>
+        </field>
+    </record>
```
Comment (yostashiro) [inline commission_disable_regenerate_agent/views/account_invoice_views.xml]: Please put this record in a separate file (sale_order_views.xml).

**Review by yostashiro — CHANGES_REQUESTED** [review]
@AungKoKoLin1997 Please add test-requirements.txt for the CI.

**Comment by yostashiro** [comment]
@AungKoKoLin1997 @kanda999 Please review and adjust this PR as necessary once https://github.com/OCA/commission/pull/387 is merged.

**Comment by AungKoKoLin1997** [comment]
We don't need to adjust this PR. This PR is also compatible with latest commission modules

**Review by yostashiro — APPROVED** [review]

**Comment by yostashiro** [comment]
@kanda999 Let's go ahead and merge https://github.com/qrtl/rmm-oca/pull/15.

@AungKoKoLin1997 Please rebase the feature branch and force-push once the above PR is merged (just to run tests).

**Comment by yostashiro** [comment]
@kanda999 Let's go ahead and merge this.

**Pull request merged**